examples/gallery/embellishments/colorbar.py

```diff
@@ -21,8 +21,8 @@
   from anchor point.
 - **n**: using normalized (0-1) coordinates, e.g. ``position="n0.4/0.8"``.
 
-Note that the anchor point defaults to the bottom left (**BL**). Append ``+h``
-to ``position`` to get a horizontal colorbar instead of a vertical one (``+v``).
+Note that the anchor point defaults to Bottom Left (**BL**). Append``+h`` to
+`position`` to get a horizontal colorbar instead of a vertical one (``+v``).
```

Comment:
```suggestion
Note that the anchor point defaults to Bottom Left (**BL**). Append``+h`` to
``position`` to get a horizontal colorbar instead of a vertical one (``+v``).
```